.gitattributes

```diff
@@ -1,9 +1,13 @@
-.editorconfig export-ignore
-.gitattributes export-ignore
-.gitignore export-ignore
-.php-cs-fixer.php export-ignore
-phpunit.xml export-ignore
-tests export-ignore
-git-hooks export-ignore
-bin/add-git-hooks export-ignore
-.github export-ignore
+/bin/ export-ignore
```

Comment:
`/bin/` is actually exported intentionally, so you can update the used public suffixes list `https://www.crwlr.software/packages/url/v2.0/mozillas-public-suffix-list`. I guess with the composer script command it's actually not working, have to fix this in the docs, but please leave the script. If you want you can ignore all the other scripts.

Reply:
Hm...OK, checked in a project where the package is installed. Can't even run the script directly in the vendor folder. Guess I'll just have to remove that possibility from the docs. So, you can leave that line (`/bin/ export-ignore`) in the `.gitattributes` file.

Reply:
Okay.